### ISSUE TITLE
[🐛 Bug]: VNC clicks not registered in Chromium-based drivers on some webpages in specific scenario

### ISSUE THREAD

**Issue by vaclavnovotny** [SeleniumHQ/docker-selenium#2392]
### What happened?

Hello!

I have a use case where I need to use the VNC of a running Selenium session to execute clicks at absolute positions in the stream. Most of the time, everything works fine. However, I encountered a specific scenario when clicking on an item in a menu that appears after hovering over it with the mouse. When the button in such a menu is clicked, nothing happens, as if the website doesn't register the click. On one page, the click registered below the menu.

Please note that this issue is not consistent—on some pages, it works, and on others, it doesn't.

Clicking in hovered menus does not work with the Chrome and Edge drivers. Interestingly, it works fine when using the Firefox driver. Therefore I suspect some issue in the chromium-based driver.

Moreover, on these problematic pages, it doesn't work even when using the NoVNC client.

Here are a few pages where this problem occurs:

https://ysoft.com/
https://printerpssw.com/
https://konicaminolta.com.au/

I understand this is a non-standard use case, but I would greatly appreciate any help, as I am at a dead end.

Thank you in advance!

### How to reproduce:

I used NuGet package 
`<PackageReference Include="Selenium.Support" Version="4.24.0" />`

And then created .NET 8 console application to just create selenium session:

```csharp
using OpenQA.Selenium.Chrome;
using OpenQA.Selenium.Remote;

var options = new ChromeOptions();
options.AddArgument("--start-fullscreen");
var driver = new RemoteWebDriver(new Uri("http://localhost:4444/wd/hub", UriKind.Absolute), options);
    
// Navigate to a website where the actual issue is
driver.Navigate().GoToUrl("https://www.ysoft.com");
    
// Break here and go to the session http://localhost:4444/ui/#/sessions
driver.Quit();
```

To reproduce the issue, simply navigate to the page, open the navigation menu, hover over a menu item, and click on a link in the displayed menu.

### Command used to start Selenium Grid with Docker (or Kubernetes)

```shell
docker run -d -p 4444:4444 -p 7900:7900 --env VNC_NO_PASSWORD=1 --shm-size="2g" selenium/standalone-chrome:4.24.0-20240907
```

### Relevant log output

```shell
No relevant errors in logs
```

### Operating System

Docker Desktop Ubuntu 22.04.1, also happens in Kubernetes running on linux

### Docker Selenium version (image tag)

4.24.0-20240907

**Comment by AkhilAppvolution** [comment]
**Subject: Issues with Clicking on Hover Menus in VNC Sessions Using Selenium with Chrome and Edge Drivers**

Hello,

I have encountered an issue similar to what was previously described regarding the use of VNC to interact with Selenium sessions for executing clicks at absolute positions in the browser. Specifically, when hovering over a menu and trying to click on an item, the click doesn't always register, or in some cases, it registers below the target. This only seems to happen in Chromium-based browsers (Chrome and Edge) while using VNC.

Like the earlier scenario, I've noticed that this problem occurs inconsistently across different websites. For example, on some pages, it works fine, while on others, the click doesn’t trigger any action. For instance, on the following websites, I've noticed these issues with hovering menus while using VNC:

In my experience, a similar issue occurs on websites such as:
- https://www.dezlearn.com/multiple-browser-windows/

However, when I run my automation without VNC—directly through the Chrome or Edge driver—the hovering and clicking behavior works perfectly. The problem only manifests when using VNC to perform these actions, especially with Chromium-based browsers.

I suspect this could be related to how the Chromium-based drivers handle hover events in VNC environments. Has anyone else experienced similar issues or found a solution to make hover-based clicks work more reliably across all browsers while using VNC?

I would appreciate any insights or suggestions!

Thank you in advance.

**Comment by vaclavnovotny** [comment]
Hello,
After a thorough investigation, we identified the issue is in the docker file itself. We created our own docker file with bare minimum things (just selenium, x11 vnc, chrome, and some utilities) and it worked.
Funnily enough, the version [4.25.0-20241010](https://hub.docker.com/layers/selenium/standalone-chrome/4.25.0-20241010/images/sha256-6ab16f7a62aec9281e0f909586dc890c4eb8a958a8044c17c93b11df48c74c8d?context=explore) somehow fixed it as well. 🤷 

So no action is needed.
Cheers!
